python_modules/dagster/dagster_tests/storage_tests/utils/event_log_storage.py

```diff
@@ -3048,16 +3048,18 @@ def _assert_matches_not_including_event_id(result, expected):
                     },
                 )
 
+                # simulate an external asset materialization for the partition... this should also make the planned
+                # materialization go away
                 storage.store_event(
```

Comment:
does this lose us any test coverage for the run-ful materialization case, should we include both?

Reply:
I'll pull it out into a separate test.